packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -181,6 +181,26 @@ function flattenTree( input = {}, prefix, token ) {
 	return result;
 }
 
+/**
+ * Resolves ref into the value it is pointing to.
+ *
+ * @param {Object} refObj      The reference we are pointing to.
```

Comment:
```suggestion
 * @param {Object} refObj      The reference in the block styles tree.
```

Nit: I have made a rule for myself to avoid using "we" in comments. It generally makes them clearer IMHO.